recipe/meta.yaml

```
@@ -178,18 +180,32 @@ outputs:
         - {{ stdlib("c") }}
       host:
         - cuda-version {{ cuda_version }}
+        - cuda-culibos-static
+        - cuda-cudart-static
       run:
         - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("libcublas-dev", max_pin="x") }}
```

```suggestion
        - {{ pin_subpackage("libcublas-dev", exact=True) }}
```
The headers that you use with the static library must match exactly.